src/transformers/pipelines/table_question_answering.py

```diff
@@ -105,9 +105,10 @@ def __init__(self, args_parser=TableQuestionAnsweringArgumentHandler(), *args, *
             else MODEL_FOR_TABLE_QUESTION_ANSWERING_MAPPING
         )
 
-        self.aggregate = bool(getattr(self.model.config, "aggregation_labels")) and bool(
-            getattr(self.model.config, "num_aggregation_labels")
+        self.aggregate = bool(getattr(self.model.config, "aggregation_labels", None)) and bool(
+            getattr(self.model.config, "num_aggregation_labels", None)
         )
+        self.type = "tapas" if hasattr(self.model.config, "aggregation_labels") else None
```

Comment:
Wondering if we can leverage `model.config.is_encoder_decoder` here

Reply:
Usually I would agree that reusing more "general" concepts is better.

The thing I want to avoid is treating `not encoder-decoder` as `tapas`. currently the code for Tapas is super highly specific, so I'd rather isolate `tapas` than isolate `tapex` if you want.

Ultimately both are OK tbh.